src/lib/y2security/security_policies/manager.rb

```diff
@@ -88,20 +91,12 @@ def enabled_policy?(policy)
         @enabled_policies.include?(policy)
       end
 
-      # Issues from all enabled policies
-      #
-      # The issues can be limited to a specific scope.
-      #
-      # @param scope [Scopes::Storage, Scopes::Bootloader, Scopes::Network, Scopes::Firewall]
-      # @return [IssuesCollection]
-      def issues(scope = nil)
-        issues_collection = IssuesCollection.new
-
-        enabled_policies.each do |policy|
-          issues_collection.update(policy, policy.validate(scope))
+      # @return [Hash<Policy, Array<Rule>>]
```

Comment:
```suggestion
      # @param config [TargetConfig]
      # @param scope [Symbol,nil] only consider rules with this scope.
      #   For example yast2-storage-ng will call this with :storage to
      #   only display rules that it can fix.
      # @return [Hash<Policy, Array<Rule>>]
```

We got rid of scope classes but I wondered why still have the argument. Now I know.